css-two-column-layout/index.html

```diff
@@ -12,14 +12,14 @@
 <body>
   <div class="container">
     <div class="row">
-      <div  class="column-one">
+      <div  class="column-full center-text">
         <h1 class="font-property head-font-property">SCALE, SPEED, AGILITY</h1>
         <p class="font-property">We've dedicated thousands of man hours to making healthcare institutions in the Helium</p>
         <p class="font-property">network run 10x more efficiently and deliver an unparalleled quality of care.</p>
       </div>
     </div>
     <div class="row">
-      <div class="column-two">
+      <div class="column-half column-margins row">
```

Comment:
No element should be both a `column` and a `row`. You likely need a `row` element as a child of this `column` element.